README.md

```diff
@@ -8,6 +8,9 @@ This is an unofficial api wrapper for TikTok.com in python. With this api you ar
 ## Sponsors
 These sponsors have paid to be placed here and beyond that I do not have any affiliation with them, the TikTokAPI package will always be free and open-source. If you wish to be a sponsor of this project check out my [GitHub sponsors page](https://github.com/sponsors/davidteather).
 
+[![Trendpop](https://trendpop-entity-assets.s3.us-west-2.amazonaws.com/customers/app/wide.png)](https://trendpop.social/careers?ref=github-davidteather-tiktokapi) | **[Trendpop](https://trendpop.social)** builds software to help creators and businesses go viral on social video platforms. Excited about building in this space? We're [hiring engineers across all roles](https://trendpop.social/careers) - shoot us a message at `founders@trendpop.social`.
```

Comment:
Can you add the `https://trendpop-entity-assets.s3.us-west-2.amazonaws.com/customers/app/wide.png` link to the `imgs` folder and just link it locally?

You might also want to add a background color to the image since this is how it shows up on GitHub light mode
![image](https://user-images.githubusercontent.com/34144122/159056939-d22789e0-50aa-4d0e-841b-c36948a32efc.png)

Reply:
ooo good catch thanks, only checked in dark mode.